_config.yml

```diff
@@ -11,14 +11,14 @@ baseurl         :
 analytics:
 
 # Social
-email        :
-facebook     :
-github       :
-gitlab       :
-google_plus  :
-instagram    :
+email        : "cesium@di.uminho.pt"
+facebook     : "https://www.facebook.com/cesiuminho"
+github       : "https://github.com/cesium"
+gitlab       : 
+google_plus  : "https://plus.google.com/103218725392069843585"
+instagram    : "https://www.instagram.com/cesiuminho/"
 linkedin     :
-twitter      :
+twitter      : "https://twitter.com/cesiuminho"
```

Comment:
All this social links should contain just the handle part. And I don't think we should keep Google+.